src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $cellgap ;
+  margin: auto;
+  width: ($cellsize * 7) + (6 * $cellgap );
+  padding: 10px;
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+}
+
+@for $i from 1 through 31 {
+  .calendar__cell-#{$i}::before {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    content: "#{$i}" ;
+    padding: auto;
```

Comment:
only `content` property is changing here - please move unchanged properieties to cell